core/src/main/java/com/netflix/conductor/core/execution/WorkflowRepairService.java

```diff
@@ -26,17 +28,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Service;
 
 /**
  * A helper service that tries to keep ExecutionDAO and QueueDAO in sync, based on the task or workflow state.
  * <p>
  * This service expects that the underlying Queueing layer implements QueueDAO.containsMessage method. This can be
- * controlled with {@link com.netflix.conductor.core.config.ConductorProperties#isWorkflowRepairServiceEnabled()}
+ * controlled with <code>conductor.app.workflow-repair-service-enabled</code>
  * property.
  */
 @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
 @Service
+@ConditionalOnProperty(name = "conductor.app.workflow-repair-service-enabled", havingValue = "true")
```

Comment:
Should this property be called `conductor.workflow-repair-service.enabled` akin to the sweeper? If not, lets change the sweeper to match this.

Reply:
I propose that enabling/disabling components within the system, properties should follow the convention:
```
conductor.component.enabled
```

For knobs to tune the system, 
- At system level
```
conductor.app.property
```
- At component level
```
conductor.component.property
```

According to this convention, this property would now become - 
```conductor.workflow-repair-service.enabled```

Thoughts?

Reply:
Agree. I will change the property name accordingly.